skops/card/_model_card.py

```diff
@@ -373,6 +374,55 @@ def add_metrics(self, **kwargs: str) -> "Card":
             self._eval_results[metric] = value
         return self
 
+    def add_permutation_importances(
+        self,
+        permutation_importances,
+        columns,
+        plot_file="permutation_importances.png",
+        plot_name="Permutation Importances",
+    ) -> "Card":
+        """Plots permutation importance and saves it to model card.
+
+        Parameters
+        ----------
+        permutation_importances : sklearn.utils.Bunch
+            Output of :func:`sklearn.inspection.permutation_importance`.
+
+        columns : str, list or pandas.Index
+            Column names of the data used to generate importances.
+
+        plot_file : str
+            Filename for the plot.
+
+        plot_name : str, or Path
+            Name of the plot.
+
+        Returns
+        -------
+        self : object
+            Card object.
+        """
+        try:
+            import matplotlib.pyplot as plt
+        except metadata.PackageNotFoundError:
+            raise ModuleNotFoundError(
+                "This feature requires matplotlib to be installed."
+            )
+
+        sorted_importances_idx = permutation_importances.importances_mean.argsort()
+        _, ax = plt.subplots()
+        ax.boxplot(
+            x=permutation_importances.importances[sorted_importances_idx].T,
+            labels=columns[sorted_importances_idx],
+            vert=False,
+        )
+        ax.set_title(plot_name)
+        ax.set_xlabel("Decrease in Score")
+        plt.savefig(plot_file)
```

Comment:
Not sure if we should care or not, but I wonder if we should check if a file by that name already exists and maybe warn the user? If they re-create the same plot, I guess it's annoying to get a warning. But what if they already happen to have a `permutation_importances.png` in the folder? Then we would just overwrite that. Really not sure...

Reply:
yes, we should have a flag, where by default we raise it the file exists, and only overwrite if the flag is True: `overwrite=True/False`